docs/Security_Considerations.md

```diff
@@ -8,10 +8,14 @@ Ethereum dApp developers looking to build on Arbitrum will generally find the ex
 
 - **Different EVM/Soldity behavior**: Arbitrum supports every EVM opcode, which means any Solidity (or Vyper, Yuul, etc.) you write will compile and run on Arbitrum. Certain operations, however, have different behavior on Arbitrum than on Ethereum; for a full list, see [Solidity Support](Solidity_Support.md). Also Arbitrum supports most — but not all — Ethereum [precompiles.](Differences_Overview.md)
 
-- **Timing Assumptions with block numbers and timestamps**: Timing assumptions one may make on layer 1 about `block.timestamp` and `block.number` (i.e., that a new block will be produced every 15 seconds on average) won't hold on Arbitrum; relaying on `block.timestamp` on L2 to measure time should only be done over large scales. For more, see [Block Numbers and Time](Time_in_Arbitrum.md)
+- **Timing Assumptions with block numbers and timestamps**: Timing assumptions one may make on layer 1 about `block.timestamp` and `block.number` (i.e., that a new block will be produced every 15 seconds on average) won't hold on Arbitrum. The the rate at which new "L2 blocks" are produced / delineated isn't reliably predictable; `block.timestamp` is a better way to measure time on Arbitrum, but even so, relying on either `block.timestamp` or `block.number` on L2 to measure time should only be done over large time scales. For more, see [Block Numbers and Time](Time_in_Arbitrum.md).
 
 - **L1-to-L2 Transactions' Address Aliases**: "Retryable tickets" are a special transaction type for sending a message from L1 to L2; if you plan on using them, it is highly recommended you read through their [documentation](L1_L2_Messages.md) and carefully test your contracts' behavior before deploying on mainnet. Of particular note: in L2 message on the receiving side, `msg.sender` not return the L1 contract, but rather it's [address alias](L1_L2_Messages.md#address-aliasing).
 
+- **L1-to-L2 Tranasaction's Ticket Creation Failure**: If you underpay for the base submission cost when attempting to create a retryable ticket, the ticket on L2 will simply not get created despite the L1 transaction being confirmed; this could — depending on what your cross-chain app is doing — be very bad. The current base submission cost can be queried directly from an Arbitrum node, and it updates once every 24 hour period; for a given update, it can increase by at most 50% of its previous value. Any amount you overpay will not be lost; it will be collected at your specified address on L2. To be safe, we highly recommend you overpay. (In a future release, the base submission cost will collected directly on L1, avoiding failure mode described above entirely.)
```

Comment:
small typo 
```suggestion
- **L1-to-L2 Transaction's Ticket Creation Failure**: If you underpay for the base submission cost when attempting to create a retryable ticket, the ticket on L2 will simply not get created despite the L1 transaction being confirmed; this could — depending on what your cross-chain app is doing — be very bad. The current base submission cost can be queried directly from an Arbitrum node, and it updates once every 24 hour period; for a given update, it can increase by at most 50% of its previous value. Any amount you overpay will not be lost; it will be collected at your specified address on L2. To be safe, we highly recommend you overpay. (In a future release, the base submission cost will collected directly on L1, avoiding failure mode described above entirely.)
```